src/lib/components/EventListElement/index.svelte

```diff
@@ -29,20 +29,26 @@
 </figure>
 
 <style>
+  h2 {
+    font-size: var(--font-300);
+  }
 	.event-tile {
 		--tw-shadow: 0 4px 6px -1px rgba(0, 0, 0, 0.1), 0 2px 4px -1px rgba(0, 0, 0, 0.06);
 		box-shadow: var(--tw-ring-offset-shadow, 0 0 #0000), var(--tw-ring-shadow, 0 0 #0000),
 			var(--tw-shadow);
 		padding: 2rem;
 		margin-top: 3rem;
 		border-radius: 0.5rem;
-		background: #f3f6f9;
+		background: var(--white);
+    display: grid;
```

Comment:
mix of tabs and spaces added to this file

Reply:
I can definitely update the max-width and may have missed some of the changes on individual pages. I went through them, but quiet a few need updated to the new variables.

Reply:
I think all of this can be adjusted to our liking after we settle on the design system and get it implemented. There are lots of little tweaks that will need to be made.
I do suggest we update the box-shadow naming too, they are confusing to me. I would prefer to go with a sizing scale for them and then we have a random one created in the EventListElement.

Reply:
I think if we can at least fix the max-width and font size in the PR that should be good for now and we can do the rest in follow ups